### DISCUSSION TITLE
Handle Udp Receive function

### DISCUSSION THREAD

**Discussion by im22361** [openthread/openthread#10297] [Q&A] [answered]
Hello,

I am trying to implement a version of the HandleUdpReceive function found here:(https://github.com/openthread/openthread/blob/0abc8d7a0db9bb75ea420388c48207737a5548e8/src/cli/cli_udp.cpp#L461). 
I am trying to read messages sent by the UDP send command, however I am struggling to find the libraries required to get this function to work, as I can't find the libraries for the OutputIp6Address function. 

Any support would be appreciated.

**Comment by jwhui** [comment, marked as answer]
`OutputIp6Address()` is implemented here:

https://github.com/openthread/openthread/blob/a54f4c485071c4909a46483e89ba527789130357/src/cli/cli_utils.cpp#L163
